pkgs/development/python-modules/recordlinkage/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, buildPythonPackage, jellyfish, numpy, pandas, scipy, scikit-learn, joblib
+, networkx, bottleneck, numexpr, pyarrow, pytest, fetchPypi }:
+buildPythonPackage rec {
+  pname = "recordlinkage";
+  version = "0.14";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-kuY2MUuwaLb228kwkJmOnnU+OolZcvGlhKTTiama+T4=";
+  };
+
+  propagatedBuildInputs = [
+    pyarrow
+    jellyfish
+    numpy
+    pandas
+    scipy
+    scikit-learn
+    joblib
+    networkx
+    bottleneck
+    numexpr
+  ];
+
+  checkInputs = [ pytest ];
+
+  pythonImportsCheck = [ "recordlinkage" ];
+
+  meta = with lib; {
+    description = "The Python Record Linkage Toolkit is a library to link records in or between data sources.";
```

Opinionated changed beside the removal of the period at the end.

```suggestion
    description = "Library to link records in or between data sources";
```